core/assets/sounds/cues/sickBeatsMegamix/data.json

```diff
@@ -1,7 +1,7 @@
 {
   "gameID": "sickBeatsMegamix",
   "gameName": "Sick Beats (Megamix)",
-  "series": "megamix",
+  "series": "side",
```

Comment:
Side games are only for games which are exclusive. Please move this back to megamix.,